stake-pool/program/src/processor.rs

```diff
@@ -1394,10 +1399,12 @@ impl Processor {
         }
 
         let stake_rent = rent.minimum_balance(std::mem::size_of::<stake::state::StakeState>());
-        if lamports < MINIMUM_ACTIVE_STAKE {
+        let stake_minimum_delegation = stake::tools::get_minimum_delegation()?;
+        let current_minimum_delegation = minimum_delegation(stake_minimum_delegation);
+        if lamports < current_minimum_delegation {
             msg!(
                 "Need more than {} lamports for transient stake to be rent-exempt and mergeable, {} provided",
-                MINIMUM_ACTIVE_STAKE,
+                current_minimum_delegation,
                 lamports
             );
             return Err(ProgramError::AccountNotRentExempt);
```

Comment:
Ooh yeah, good point. How about `StakeError::InsufficientDelegation`?

Reply:
This was a tricky one -- the API for `increase` takes in the final amount that the destination will increase by, so for example, if I want to add 2 SOL to a validator, I pass in 2 (billion).  However, because the rent-exempt amount is not part of the delegation, the reserve actually needs to split off 2 + rent-exemption. The pool withdraws the rent-exemption back to the reserve during the update.

All this to say, how about "Need more than {} lamports for transient stake to meet minimum delegation requirement"?